frontend/src/App.js

```diff
@@ -0,0 +1,59 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
+import { ToastContainer } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
+import Home from './pages/Home'
+import NavBar from './components/NavBar'
+import Login from './pages/Login'
+import Register from './pages/Register'
+import Footer from "./components/Footer";
+import UserHome from "./pages/UserHome";
+import PostDetail from "./pages/PostDetail";
+import FavoritePost from "./pages/FavoritePost";
+import Profile from "./pages/Profile";
+// import { useEffect } from 'react'
+// import { useSelector, useDispatch } from 'react-redux'
+// import Spinner from './components/Spinner'
+// import { getAllPosts, reset } from './features/posts/postSlice'
+function App() {
+
+  // const dispatch = useDispatch()
+
+  // const { posts, isLoading, isError, message } = useSelector((state) => state.posts)
+
+  // useEffect(() => {
+  //   if (isError) {
+  //     console.log(message)
+  //   }
+
+  //   dispatch(getAllPosts())
+
+  //   return () => {
+  //     dispatch(reset())
+  //   }
+  // }, [isError, message, dispatch])
+
+  // if (isLoading) {
+  //   return <Spinner />
+  // }
```

Comment:
Commented out code should not be merged. If you need to recover it after deletion, you can always look at the commit where it was removed to see it.